app/views/subscriptions/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<br>
+
+<div class="container">
+  <% @subscriptions.each do|subscription|%>
+   <p><strong><%= subscription.name%></strong>
+    <br><%= subscription.content%>
+    <br>Number of lessons: <%= subscription.package_size%>
+    <br><%= subscription.durability%>
+    <br><%= subscription.amount%> IDR
```

Comment:
do we have to pass IDR here or will out monetize gem automatically generate the currency to display?

Reply:
not sure, ill find out and will update